src/promptflow-tracing/promptflow/tracing/_openai_utils.py

```diff
@@ -155,24 +155,27 @@ def _get_completion_tokens_for_chat_api(self, output, enc):
 
     def get_openai_metrics_for_completion_api(self, inputs, output):
         metrics = {}
-        enc = self._get_encoding_for_completion_api(self._try_get_model(inputs, output))
-        metrics["prompt_tokens"] = 0
-        prompt = inputs.get("prompt")
-        if isinstance(prompt, str):
-            metrics["prompt_tokens"] = len(enc.encode(prompt))
-        elif isinstance(prompt, list):
-            for pro in prompt:
-                metrics["prompt_tokens"] += len(enc.encode(pro))
-        if isinstance(output, list):
-            if IS_LEGACY_OPENAI:
-                metrics["completion_tokens"] = len(output)
+        try:
+            enc = self._get_encoding_for_completion_api(self._try_get_model(inputs, output))
+            metrics["prompt_tokens"] = 0
```

Comment:
I have created a constant class `TokenKeys` for these 3 tokens, , consider using those instead of hard coding, no need to do it in this pr tho.

Reply:
Will fix in other PR.